src/components/ScrollTable/ScrollTable.less

```diff
@@ -0,0 +1,15 @@
+.lucid-ScrollTable {
+	border: @Table-cell-border;
```

Comment:
Any time we use variables from other components, we need to hoist them up in to `variables.less`